Ppap.scala

```diff
@@ -2,7 +2,7 @@ object Main extends App{
   import scala.util.Random
   
   def ppap(pp: List[String] = Nil): Unit = pp match {
-    case "pen" :: "pineapple" :: "apple" :: "pen" :: _ => println(pp.reverse.mkString + " Ah-! Pen Pineapple Apple Pen")
+    case "pen" :: "pineapple" :: "apple" :: "pen" :: _ => println(pp.reverse.mkString + " Ah-! pen-pineapple-Apple-pen!")
     case _ => ppap((if(Random.nextBoolean) "Pen" else if(Random.nextBoolean) "Pineapple" else "apple") :: pp)
```

Comment:
Note that the current your code is endless recursion since `"pen"` doesn't match `"Pen"` and `"pineapple"` doesn't match `"Pineapple"`.